codes.md

```diff
@@ -352,6 +369,190 @@ reasoning to the given organization that they cannot infer.
+      - [Session 021](https://osf.io/v4nmf/) - uses "buckets" and
+        "underneath", draws indented tree first, node-link second.. with the
+second uses language like 'hierarchy', 'contains' and 'includes'
+        - "When you said I had two folders, I basically created two buckets withthese boxes. Underneath each box I listed what was included in those folders."
+        - "To me they’re buckets. I kind of think of a folder as “oh here, here’s a bucket and here’s what’s in the bucket."
+        - (second drawing) "So I did label the folders and explicitly labeled here's the first folder, and here's the second folder, and here's what isincluded in this folder and I grouped them together, with arrows to say this folder contains this group of things, and these things are text or image files, and then this folder includes this set of things, which also includes another folder, and that folder includes this set of things, so I kind of wanted to show visually that Folder 1 is kind of like a container that included, you know five things, whereas Folder 2 was a container that contains seven things with one of them actually containing two more things, so I was kind of showing that hierarchy, a little bit and and trying to help visualize these things as being together in a group, by boxing them around."
+    - Network as Node-Link / Physical:
+       - [Session 008](https://osf.io/wb4r5/) - uses word "connections" but
+         also "wires", draws network
+         - "Initially I was just picturing in my mind an actual power station that is connected to another building, or something. "
+         - "These represent wires."
+    - Network as Node-Link:
+       - [Session 020](https://osf.io/5rsjc/) - uses "node", "connect",
+         "graph" and  draws node-link
+         - " I made each power station a separate node and then connected the nodes because I’m presuming they can connect the power stations."
+         - "And then off each power station node, if it was residential I indicated that by an arrow that would indicate the number of residential spots."
+         - "A graph basically."
+         - "No, I haven’t seen a data set like that before, but this kind of graphing is kind of the way we think about problems and what we do at work.It tends to be kind of the default model I use for mathematical problems. I’ll always make it a graph problem, or frequently will. "
+    - "Folder Structure" as Enclosure/Nesting with Links (Hybrid):
+       - [Session 009](https://osf.io/es7w6/) - uses "inside"/"outside" to describe,
+         draws nesting. Claims this is a "folder structure" repeatedly. Never
+says "tree"
+    - ?? as containment/enclosure: KATY/ALEX
```

Comment:
Collections/Groups/Belonging maybe? The participant never used (clear) language to define or indicate an overall abstraction. They only used squares for the folders and containment within the folders. It looks a lot like [Session 009](https://osf.io/es7w6/)

Reply:
Yeah that's why I put it right under Session 009. Maybe I'll list it as "abstraction unclear".